### PR TITLE
Add divide function and test

### PR DESCRIPTION
**Pull request by TingnaW**
Resolves # 2

### CODE REVIEW

**Comment by TingnaW** [comment]
# 2

File: pythoncalculator/divide.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
+    return x / y
```
Comment (dr-slburg) [inline pythoncalculator/divide.py]: add new line

File: pythoncalculator/divide.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```
Comment (dr-slburg) [inline pythoncalculator/divide.py]: change z to y

File: tests/test_divide.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+from pythoncalculator import divide
+
+
+def test_divide():
+    assert divide(10, 2) == 5
```
Comment (dr-slburg) [inline tests/test_divide.py:5]: add new line

**Review by dr-slburg — APPROVED** [review]

**Comment by TingnaW** [comment]
Thx

**Pull request merged**